pom.xml

```diff
@@ -215,7 +215,7 @@
     <dependency>
       <groupId>xom</groupId>
       <artifactId>xom</artifactId>
-      <version>1.1</version>
+      <version>1.3.5</version>
```

Comment:
Should be `<version>[1.3.5,)</version>`

Otherwise, you are forcing downstream projects to use a specific version of XOM (or to configure POM exclusions).

Reply:
> Should be `<version>[1.3.5,)</version>`

Thanks — changed it to that